src/hotspot/os/posix/os_posix.cpp

```diff
@@ -1332,7 +1332,7 @@ static jlong millis_to_nanos_bounded(jlong millis) {
 
 static void to_abstime(timespec* abstime, jlong timeout,
                        bool isAbsolute, bool isRealtime) {
-  DEBUG_ONLY(int max_secs = MAX_SECS;)
+  DEBUG_ONLY(int64_t max_secs = MAX_SECS;)
```

Comment:
Actually I think this should be a time_t.

Reply:
agree.